src/main/java/com/cloudogu/scm/mirror/api/MirrorAccessConfigurationToConfigurationDtoMapper.java

```diff
@@ -40,6 +41,7 @@
 import javax.inject.Inject;
 import javax.inject.Provider;
 
+import static com.cloudogu.scm.mirror.MirrorConfiguration.*;
```

Comment:
Normally we avoid star imports... Wonder how you got there. `UsernamePasswordCredential` is imported differently.
```suggestion
import com.cloudogu.scm.mirror.MirrorAccessConfiguration.CertificateCredential;
```